Formula/lxc.rb

```diff
@@ -1,10 +1,8 @@
-require "language/go"
-
 class Lxc < Formula
   desc "CLI client for interacting with LXD"
   homepage "https://linuxcontainers.org"
-  url "https://github.com/lxc/lxd/archive/lxd-2.12.tar.gz"
-  sha256 "8cc4143be017c1f6c70489c5745903fa8798acb7ed03f743b543689746eceb9f"
+  url "https://linuxcontainers.org/downloads/lxd/lxd-2.12.tar.gz"
+  sha256 "439bdadc7c63c523999a8d26b274a5c1c397d897197b43df8ce4b225a75585ff"
```

Comment:
Given the extension nature of the changes (even if it is the same version), we probably want a `revision 1` line after the `sha256` line.